pkg/asset/rhcos/image.go

```diff
@@ -52,101 +55,107 @@ func (i *Image) Dependencies() []asset.Asset {
 func (i *Image) Generate(ctx context.Context, p asset.Parents) error {
 	if oi, ok := os.LookupEnv("OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE"); ok && oi != "" {
 		logrus.Warn("Found override for OS Image. Please be warned, this is not advised")
-		*i = Image(oi)
+		*i = *MakeAsset(oi)
 		return nil
 	}
 
 	ic := &installconfig.InstallConfig{}
 	p.Get(ic)
 	config := ic.Config
-	osimage, err := osImage(ctx, config)
+	osimageControlPlane, osImageCompute, err := osImage(ctx, config)
```

Comment:
hmm..computes can be an array right? shouldn't this be an array?

Reply:
They are an array but the installer only supports 1 element currently.